src/legacy/server/index_patterns/service/lib/field_capabilities/field_capabilities.test.ts

```diff
@@ -72,7 +68,7 @@ describe('index_patterns/field_capabilities/field_capabilities', () => {
   describe('calls `readFieldCapsResponse`', () => {
     it('passes exact es response', async () => {
       stubDeps({
-        esResponse: footballs[0]
+        esResponse: footballs[0],
```

Comment:
@lukeelmers These test files are completely filled with errors because the way the code is called here is totally different from the actual code- suggestions?